packages/mui-component-mapper/src/files/select.js

```diff
@@ -44,15 +113,66 @@ Select.propTypes = {
   isSearchable: PropTypes.bool,
   options: PropTypes.arrayOf(PropTypes.shape({ value: PropTypes.any.isRequired, label: PropTypes.node.isRequired })).isRequired,
   description: PropTypes.node,
-  FormFieldGridProps: PropTypes.object
+  FormFieldGridProps: PropTypes.object,
+  value: PropTypes.any,
+  isClearable: PropTypes.bool,
+  classNamePrefix: PropTypes.string,
+  isMulti: PropTypes.bool,
+  loadingMessage: PropTypes.node,
+  onInputChange: PropTypes.func,
+  isFetching: PropTypes.bool,
+  noOptionsMessage: PropTypes.func,
+  closeMenuOnSelect: PropTypes.bool,
+  onChange: PropTypes.func.isRequired,
+  hideSelectedOptions: PropTypes.bool,
+  required: PropTypes.bool,
+  onFocus: PropTypes.func,
+  onBlur: PropTypes.func,
+  TextFieldProps: PropTypes.object,
+  inputProps: PropTypes.object
```

Comment:
What is the difference between input props and TextField props?

Reply:
They are different components. TextField controls the labels colors, helper text, etc, but the input props control the actual input.

Its been done based in MUI docs.